bin/rtl-goldcheck.sh

```diff
@@ -89,7 +89,7 @@ if [ "$1" == "amber" ]; then
     bash -c 'set -x; source tmp-amber-updates.sh'
 
 elif [ "$1" == "onyx" ]; then
-    flags="$flags --rv --sparse-cgra --sparse-cgra-combined"
+    flags="$flags --include-sparse"
```

Comment:
Wow you found and fixed my goldcheck script! Thanks! -Steve

Reply:
> Wow you found and fixed my goldcheck script! Thanks! -Steve

...ditto for my daemon files and `gen-rtl` script, this will save me future headaches I am sure...